MapboxCoreNavigation/BorderCrossingInfo.swift

```diff
@@ -0,0 +1,23 @@
+
+import Foundation
+import MapboxNavigationNative
+import MapboxDirections
+
+extension AdministrationRegion {
+    init(_ adminInfo: RouteAlertAdminInfo) {
+        self.init(countryCode: adminInfo.iso_3166_1, countryCodeAlpha3: adminInfo.iso_3166_1_alpha3)
+    }
+}
+
+/**
+ `BorderCrossingInfo` encapsulates a border crossing, specifying crossing region codes.
+ */
+public struct BorderCrossingInfo {
+    public let from: AdministrationRegion
+    public let to: AdministrationRegion
```

Comment:
Is it expected that there will be more information about the border crossing, or could it be more efficiently represented as a tuple?

Reply:
To be honest: I don't know. At least now having it as `struct` is justified by introducing a convenience initializer which makes it easier to use than a `tuple`.